src/main/java/frc/robot/Robot.java

```diff
@@ -60,6 +60,11 @@ public void robotInit() {
         // autoChooser.setDefaultOption(name, object);
         // autoChooser.addOption(name, object);
 
+        //set up command logging
+        CommandScheduler.getInstance().onCommandInitialize((command) -> DriverStationLogger.logToDS("Starting to run " + command.getName().toLowerCase()));
+        CommandScheduler.getInstance().onCommandInterrupt((command) -> DriverStationLogger.logToDS("Interrupting " + command.getName().toLowerCase()));
+        CommandScheduler.getInstance().onCommandFinish((command) -> DriverStationLogger.logToDS(command.getName() + " is finished"));
+
```

Comment:
//set up command logging
        CommandScheduler.getInstance().onCommandInitialize((command) -> DriverStationLogger.logToDS("Starting to run " + command.getName().toLowerCase()));
        CommandScheduler.getInstance().onCommandInterrupt((command) -> DriverStationLogger.logToDS("Interrupting " + command.getName().toLowerCase()));
        CommandScheduler.getInstance().onCommandFinish((command) -> DriverStationLogger.logToDS(command.getName() + " is finished"));
did you ment to put those in periodic? not robot init... 
also you dont need (command), use:
command -> ...

Reply:
I did not mean to put it in periodic. The command scheduler calls those lambdas every time a command start/interrupt/finishes.
see [WPILib example](https://github.com/wpilibsuite/allwpilib/tree/master/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/schedulereventlogging)